src/ui/api/project.go

```diff
@@ -140,14 +140,10 @@ func (p *ProjectAPI) Head() {
 		return
 	}
 
-	userID := p.ValidateUser()
+	_ = p.ValidateUser()
```

Comment:
Should we check user first? And seems "project.Public == 1" in the if statement is not needed anymore.